jenkins-integration/beaker/gatling/10_start_gatling_scenario.rb

```diff
@@ -0,0 +1,35 @@
+test_name "Start gatling scenario"
+
+simulation_id = ENV['GATLING_SIMULATION_ID']
+gatling_scenario = ENV['GATLING_SCENARIO']
+sut_hostname = ENV['GATLING_SUT_HOSTNAME']
+
+if not simulation_id
+  abort "GATLING_SIMULATION_ID environment variable required"
+elsif not gatling_scenario
+  abort "GATLING_SCENARIO environment variable required"
+elsif not sut_hostname
+  abort "GATLING_SUT_HOSTNAME environment variable required"
+end
+
+repo_url = "https://github.com/puppetlabs/gatling-puppet-load-test.git"
+repo_path = "/root/gatling-puppet-load-test_#{simulation_id}"
+
+sbt_path = "#{repo_path}/simulation-runner"
+
+# Sets the environment variables and runs sbt
+sbt_command = "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/#{gatling_scenario} PUPPET_GATLING_SIMULATION_ID=#{simulation_id} PUPPET_GATLING_MASTER_BASE_URL=#{sut_hostname} sbt run"
+
+# To keep log/result files separate on the gatling machine
+step "Clone gatling-puppet-load-test into new folder for this scenario"
+on(gatling, "git clone #{repo_url} #{repo_path}")
+
+step "Start gatling with scenario on gatling machine"
+# This seems like an ugly but accepted way of starting background tasks over ssh
+nohup_cmd = "nohup bash -c 'cd #{sbt_path} && #{sbt_command}' &> #{sbt_path}/sbt.out < /dev/null &"
+on(gatling, nohup_cmd)
```

Comment:
Yeah, for now I think we should focus on the use cases around shorter-term performance and regression testing.  So, assuming that most of these runs will be done, say, nightly or weekly, and will only run for a few hours.

We'll expand out from there to deal with the 'months-long' runs.  For those we may have to have a different pattern as I don't know how Jenkins will do with a run that goes for that long.  But the ability to have Jenkins collect artifacts and update graphs, etc., at the end of a run is a big enough win to where I think that should be our first step.

So, yeah, JJB seems like the way to go.  Maybe also updating the dev environment tooling to install the sbt plugin for jenkins?